frontend/src/components/associations/elections/edit/Edit.tsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const AssociationEditElection = ({ association }) => {
+  return <p>View vote</p>;
```

Comment:
EditElection but View vote?

Reply:
I don't care obviously but that's weird

Reply:
It’s dummy text, I’ll disappear in the next PRs